cdap-data-fabric/src/test/java/co/cask/cdap/spi/data/common/CachedStructuredTableRegistryTest.java

```diff
@@ -0,0 +1,96 @@
+/*
+ * Copyright © 2019 Cask Data, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not
+ * use this file except in compliance with the License. You may obtain a copy of
+ * the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+ * License for the specific language governing permissions and limitations under
+ * the License.
+ */
+
+package co.cask.cdap.spi.data.common;
+
+import co.cask.cdap.spi.data.TableAlreadyExistsException;
+import co.cask.cdap.spi.data.table.StructuredTableRegistry;
+import co.cask.cdap.spi.data.table.StructuredTableRegistryTest;
+import org.junit.Assert;
+import org.junit.Test;
+
+import java.io.IOException;
+
+/**
+ *
+ */
+public abstract class CachedStructuredTableRegistryTest extends StructuredTableRegistryTest {
+  /**
+   * @return non-cached version of StructuredTableRegistry
+   */
+  protected abstract StructuredTableRegistry nonCachedStructuredTableRegistry();
```

Comment:
better call this `getNonCachedStructuredTableRegistry()` to be consistent with `getStructuredTableRegistry()`.

Reply:
Done